.github/workflows/system-tests-enterprise.yml

```diff
@@ -194,12 +194,9 @@ jobs:
         
         # Get the tested mlrun version
         echo "unstable_version_prefix=$( \
-          cd /tmp && \
-          git clone --single-branch --branch ${{ steps.current-branch.outputs.name }} https://github.com/mlrun/mlrun.git mlrun-upstream 2> /dev/null && \
-          cd mlrun-upstream && \
-          cat automation/version/unstable_version_prefix && \
-          cd .. && \
-          rm -rf mlrun-upstream)" >> $GITHUB_OUTPUT
+          curl https://raw.githubusercontent.com/mlrun/mlrun/${{ steps.current-branch.outputs.name }}/automation/version/unstable_version_prefix \
+        )" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
         >> $GITHUB_OUTPUT
```

Reply:
see line 196, it opens (" so the closure is at 198